controllers/article-controller.js

```diff
@@ -0,0 +1,45 @@
+const {
+    selectArticles,
+    selectArticleById,
+    setArticleById,
+  } = require("../models/article-model.js");
+
+
+const getArticles = (request, response) => {
+  selectArticles().then(({ rows: articles }) => {
```

Comment:
Please keep your de-structuring consistent across your functions. I prefer the way you've done it for selectArticleById below (de-structured the article from rows in the model before returning back up to the controller.)